src/coreclr/vm/stubgen.cpp

```diff
@@ -1212,10 +1212,10 @@ void ILCodeStream::EmitCALLVIRT(int token, int numInArgs, int numRetArgs)
     WRAPPER_NO_CONTRACT;
     Emit(CEE_CALLVIRT, (INT16)(numRetArgs - numInArgs), token);
 }
-void ILCodeStream::EmitCALLI(int token, int numInArgs, int numRetArgs)
+void ILCodeStream::EmitCALLI(int token, int numInArgs, int numRetArgs, bool explicitThis)
```

Comment:
I think it would make more sense to adjust `numInArgs` on the caller side and leave this method as is.

Reply:
Done